conf/upgrade.yaml.template

```diff
@@ -5,6 +5,10 @@ UPGRADE:
   TO_VERSION: "6.9"
   # Satellite, Capsule hosts RHEL operating system version.
   OS: "rhel7"
+  # The workflow Broker should use to checkout a to-be-upgraded Satellite
+  SATELLITE_DEPLOY_WORKFLOW: deploy-satellite-upgrade
```

Comment:
This is not needed as it is going to be exported as
`ROBOTTELO_server__deploy_workflows__product`

Reply:
Even better!